File: Assignment___43/Assignment_1.c
#include<stdio.h>


int WhiteSpace(char *str)
{
    static int iCount = 0;
    if(*str == '\0')
    {
        return 0;
    }
    else 
    {
        if(*str == ' ')
        {
            iCount++;
        }
        WhiteSpace(str+1);
    }   
    
    return iCount;
}

int main()
{
    int iRet = 0 ;
    char arr[20];

    printf("Enter String : ");
    scanf(" %[^\n]s",arr);

    iRet = WhiteSpace(arr);
    printf("%d\n",iRet);

    return 0;
}

File: Assignment___55/Assignment_1.java
import java.util.*;



class MyArray
{
    private char Arr[];

    public MyArray(int iSize)
    {
        Arr = new char[iSize];
    }

    public void Accept()
    {
        Scanner sobj = new Scanner (System.in);
        System.out.println("Enter "+ Arr.length+" Elements : ");

        for(int i = 0 ; i < Arr.length; i++)
        {
            Arr[i] = sobj.next().charAt(0);
        }
    }

    public void Display()
    {
        for(int i = 0 ; i < Arr.length; i++)
        {
            System.out.print(Arr[i]+"\t");
        }
        System.out.println();
    }

    public void ArrayReplace()
    {
        for(int i = 0; i < Arr.length; i++ )
        {
            if(Arr[i] >= 'A' && Arr[i] <= 'Z')
            {
                Arr[i] = (char)(Arr[i]+32);
            }
        }
    }
}
class Assignment_1
{

    public static void main(String A[])
    {
        Scanner sobj = new Scanner (System.in);
        int iRet = 0 ;

        System.out.println("Enter Size of first Array : ");
        int iValue1 = sobj.nextInt();

        MyArray Mobj1 = new MyArray(iValue1);
        Mobj1.Accept();

        System.out.println("Elements of Array are : ");

        Mobj1.Display();
        Mobj1.ArrayReplace();

        System.out.println("Array after replacement : ");
        Mobj1.Display();

    }
}
